build.gradle

```diff
@@ -574,6 +574,7 @@ check.dependsOn integrationTest
 
 dependencies {
     implementation "org.opensearch.plugin:transport-netty4-client:${opensearch_version}"
+    implementation "org.opensearch.plugin:telemetry-otel-client:${opensearch_version}"
```

Comment:
We should never use this artifact , it is an sad oversight we publish it (@Gaganjuneja fyi), the plugins should only be imported and used as ZIP archives.

Reply:
+1. If it is absolutely needed, can we have it as a testImplementation dependency?

Reply:
You are right. Till the time https://github.com/opensearch-project/security/issues/4010 is not closed we can go ahead with having it as a testImplementation dependency.